amundsen_application/static/js/components/TableDetail/EditableSection/tests/index.spec.tsx

```diff
@@ -65,5 +66,18 @@ describe("EditableSection", () => {
       const { wrapper } = setup(null, child);
       expect(wrapper.childAt(1).text()).toBe(child);
     });
+
+    it("renders button when readOnly=false", () => {
+      expect(wrapper.find(".btn").length).toEqual(1);
+    });
+
+    it("renders does not add button when readOnly=true", () => {
+      const { wrapper } = setup({readOnly: true}, <TagInput/>);
+      expect(wrapper.find(".btn").length).toEqual(0);
+    });
+
+    it( 'renders modifies title to have no underscores', () => {
```

Comment:
This unit test suffers from leveraging knowledge of how `convertText` works. Ideally the component is only concerned with checking that it called `convertText` with the correct parameters. If you aren't familiar with mocking functions in Jest,a fter this is merged I think I'll update this test in particular as an example that you can reference in the future.

Reply:
ok, yeah I just copied the test style from Flag. I am not familiar with jest. Will be a good learning experience to see that example in the future, thanks!